### PR TITLE
updates dev dependencies to support svelte-5 release candidate. Have …

### PR DESCRIPTION
**Pull request by travisdmathis**
…tested and everything works fine. please enable svelte 5

### CODE REVIEW

**Comment by joshnuss** [comment]
Hi,

Does the version be specified as 5.0.0 or will that not match RC releases?

**Comment by travisdmathis** [comment]
> Hi,
> 
> Does the version be specified as 5.0.0 or will that not match RC releases?

^5.0.0 will not resolve hyphenated names so it has to be specific.. once they drop the next tag you can change it to ^5.0.0.

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
   },
   "peerDependencies": {
     "@stripe/stripe-js": "^3",
-    "svelte": "^3 || ^4"
+    "svelte": "^3 || ^4 || ^5.0.0-next.120"
```
Comment (gagansuie) [inline package.json:41]: looks like latest is currently [5.0.0-next.173](https://www.npmjs.com/package/svelte/v/5.0.0-next.173)

**Comment by joshnuss** [comment]
The hold up is that this version will constantly change.

Maybe it should just be changed to be greater than 3, ie `>= 3`

File: package.json (hunk 1 above)
Comment (evdama) [reply package.json:41]: Svelte 5 is now released.

**Comment by joshnuss** [comment]
Issue is resolved in latest version.

**Pull request closed without merging**